### PR TITLE
testing doc changes

### PR DESCRIPTION
**Pull request by lalchand12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,16 @@
...
-Terrascan detects security vulnerabilities and compliance violations across your Infrastructure as Code. Mitigate risks before provisioning cloud native infrastructure. Run locally or integrate with your CI\CD.
+## Introduction
 
+Terrascan is a static code analyzer for Infrastructure as Code. Terrascan allow you to:
+
+- Detect security vulnerabilities and compliance violations across your Infrastructure as Code.
+- Mitigate risks before provisioning cloud native infrastructure.
+- Offers flexibility to run locally or integrate with your CI\CD.
+
+## Resources
```
Comment (amirbenv) [inline README.md:20]: Would it look better to have resources and features as ### instead of ##?

File: README.md [hunk 2]
```diff
@@ -70,52 +99,37 @@ Flags:
...
 ### Kubernetes
-In Kubernetes yamls, you can tell terrascan to skip rules by adding an annotation as seen in the snippet below.
+In Kubernetes yamls, you can configure Terrascan to skip rules by adding an annotation as seen in the snippet below.
 
 ![k8s](https://user-images.githubusercontent.com/74685902/105115885-834a5d00-5a7e-11eb-9190-e8b64d77c5ac.png)
 
-### Broad Rule Suppression
-Use our config file to manually pick which rules should be applied or suppressed from the entire scan. This is suitable for edge use cases. Please use in-file suppression to specify resources that shouldn't be tested against particular rules. This ensures that the rules are skipped only for particular resources, rather than all of the resources.
+### Global Rule Suppression
+Use the Terrascan config file to manually select the rules should be excluded or suppressed from the entire scan. This is suitable for edge use cases.
```
Comment (amirbenv) [inline README.md:123]: i think we need to change "excluded or suppressed" to "**included** or suppressed"

**Review by amirbenv — COMMENTED** [review]
Thanks for the edits and clarifications!

**Comment by lalchand12** [comment]
This is a work in progress, will update when done

**Pull request closed without merging**